strongswan/config.sls

```diff
@@ -12,13 +12,15 @@
 {%- set connections = {} %}
 {%- set conn_default = {} %}
 
-{%- for k, v in conn.items() %}
-  {%- if k == '_default' %}
-    {%- do conn_default.update(v) %}
-  {% else %}
-    {%- do connections.update({k:v}) %}
-  {%- endif %}
-{%- endfor %}
+{%- if conn %}
```

Comment:
@daks, I took a quick glimpse of the PR and fail to see why you need to add this conditional here, as [the default value for `conn`, if not set is an empty value](https://github.com/saltstack-formulas/strongswan-formula/blob/e99c764c9f529425e677ce22cb83ec6bd5a25f1b/strongswan/config.sls#L11) and, therefore, this loops is skipped. Am I missing something?

Also, I think that probably is better to set a default value for `conn` in `defaults.yaml` and change that same line I mention above accordingly, wdyt?

Reply:
I was surprised by this bug too thinking that `{%- set conn = salt['pillar.get']('strongswan:conn', {}) %}` was sufficient. But it's not :) The first commit implements a test which proves this bug.

I prefer to think that `defaults.yaml` is used for 'core' parameters to configure the formula (like config directory, file names...) and the distinction between OS and releases.
I don't think that default parameters to be passed to the managed software itself should be put in `defaults.yaml`.

Moreover the `conn` configuration is what the user need to define, it has no sense to have a 'default' value which be useless for anyone.